site/content/extend/customization/webapp.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Web App"
+date: 2018-05-20T11:35:32-04:00
+weight: 1
+subsection: Customizing Mattermost
+---
+
+Customizations to the Mattermost Web App can be performed in cases where you need to customize branding, alter localization strings, or fulfill security requirements that are not immediately offered out-of-the-box.  
+  
+## Customization steps 
+With that in mind, customizing and deploying your Mattermost Web App can be done in a few steps:  
+
+1. Fork the [mattermost-webapp](https://github.com/mattermost/mattermost-webapp) repository, and then clone your fork in your local environment.  
+```
+git clone https://github.com/<yourgithubusername>/mattermost-webapp
+```
+
+2. Create a separate branch to perform customization tasks in, as it's not recommended to perform them in the `master` branch (more details about that in the next section regarding rebasing).  
```

Comment:
"Create a separate branch to perform customization tasks in" - "in" what?

Reply:
Ah, I was under the impression that was a thing people said in English, the intent was to tell people to create a branch for all their customization-related tasks.

I'll then reword this to `Create a separate branch for your customized version`, does that sound good?

Reply:
Makes sense now, I read it wrong the first time - works either way then.